lib/transformReadonlyNonEmptyArray.test.ts

```diff
@@ -0,0 +1,59 @@
+import * as ReadonlyNonEmptyArray from 'fp-ts/ReadonlyNonEmptyArray';
+import { expectType } from 'ts-expect';
+import * as zod from 'zod';
+
+import { transformReadonlyNonEmptyArray } from './transformReadonlyNonEmptyArray';
+
+describe('transformReadonlyNonEmptyArray()', () => {
+  test('adds issue to refinement context when array is empty', async () => {
+    const addIssue = jest.fn();
+    const ctx: zod.RefinementCtx = {
+      addIssue,
+      path: [''],
+    };
+
+    transformReadonlyNonEmptyArray([], ctx);
+
+    expect(addIssue).toHaveBeenCalledTimes(1);
+    expect(addIssue.mock.calls[0][0]).toEqual({
+      code: zod.ZodIssueCode.custom,
+      message: 'Array must not be empty',
+      fatal: true,
+    });
+  });
+
+  test('transforms non-empty array to fp-ts NonEmptyArray type', async () => {
```

Comment:
[question] Should this be:

```suggestion
  test('transforms non-empty array to fp-ts ReadonlyNonEmptyArray type', async () => {
```

Reply:
Fixed https://github.com/causaly/zod-non-empty-array/pull/1/commits/84528c9202562825bda95d368f07e2b705b86ea8